### PR TITLE
Add commands to rbe.md

### PR DESCRIPTION
**Pull request by zanderso**

### CODE REVIEW

**Review by chinmaygarde — APPROVED** [review]

File: docs/rbe/rbe.md [hunk 1]
```diff
@@ -116,6 +157,10 @@ builds.
 
 The status of the local RBE proxy can be queried with the following command
 
+```
+buildtools/mac-arm64/reclient/reclientreport
```
Comment (chinmaygarde) [inline docs/rbe/rbe.md:161]: This file doesn't seem to exist on my machine.

Comment (zanderso) [reply docs/rbe/rbe.md:161]: Hmm. Do you see it under `//flutter/buildtools/...` ?

Comment (chinmaygarde) [reply docs/rbe/rbe.md:161]: Ah, wrong buildtools. Yeah, I see it there. But that just seems to generate a tarball with logs that are pretty hard to parse. Not sure we should include it as a debugging aid.

Comment (zanderso) [reply docs/rbe/rbe.md]: Whoops. It should have been `reproxystatus` not `reclientreport`.

Comment (chinmaygarde) [reply docs/rbe/rbe.md]: That always reports that reproxy is not running. Presumably because et starts and stops it as needed?
